src/i18n/nl/index.ts

```diff
@@ -185,10 +185,10 @@ const nl = {
       BADGES: 'Badges',
     },
     EMPTY_CREDENTIALS: {
-      TITLE: '',
-      SUBTITLE: '',
+      TITLE: 'Het is hier een beetje stil',
+      SUBTITLE: 'Waarom voeg je geen nieuwe credentials toe om te starten met je digitale "me"?',
```

Comment:
Ok hier: `Voeg credentials toe om je digitale "me" te starten.

Reply:
imo `'Waarom voeg je geen nieuwe credentials toe om te starten met je digitale "me"?'` captures the same kind of informalness/playfulness as `'Why not add some credentials to start your new digital me?'`